.drone.star

```diff
@@ -1140,11 +1147,14 @@ def coreApiTests(ctx, part_number = 1, number_of_parts = 1, storage = "ocis", ac
                              "DIVIDE_INTO_NUM_PARTS": number_of_parts,
                              "RUN_PART": part_number,
                              "ACCEPTANCE_TEST_TYPE": "core-api",
-                             "EXPECTED_FAILURES_FILE": expectedFailuresFile,
+                             "EXPECTED_FAILURES_FILE": expected_failures_file,
                              "UPLOAD_DELETE_WAIT_TIME": "1" if storage == "owncloud" else 0,
                              "OCIS_WRAPPER_URL": "http://ocis-server:5200",
+                             "WITH_REMOTE_PHP": get_remotephp_config(ctx),
                          },
                          "commands": [
+                             # merge the expected failures
+                             "" if get_remotephp_config(ctx) else "cat %s/expected-failures-without-remotephp.md >> %s" % (test_dir, expected_failures_file),
```

Comment:
can't we do with inside script using env

Reply:
let's keep it as it is
https://github.com/owncloud/ocis/pull/10078#discussion_r1808535432